### DISCUSSION TITLE
支持多source join 么？

### DISCUSSION THREAD

**Discussion by LeeMo2K10** [apache/seatunnel#5015] [Q&A]
看到说2.3.1 取消了 join， 那多source 有啥场景呢？

**Comment by xduooo** [comment]
Same question, how can i do like this
```
transform {
  Sql {
    result_table_name = "fake_final"
    query = "select fake1.id,fake1.age,fake2.name from fake1 join fake2 on fake1.id=fake2.id"
  }
}

sink {
  Console {
    source_table_name = "fake_final"
  }
}
```

**Reply by liuyhl** [reply]
same question

**Comment by liunaijie** [comment]
No, we not support `join`.

**Reply by cascat0** [reply]
在source的query参数中写JOIN不可以吗？

**Reply by liunaijie** [reply]
> 在source的query参数中写JOIN不可以吗？

the source query is submit to source connector directly, so you can write join in source (if the source support)
What I mentioned ` we not support  join`, it means we can use join in `Sql` transform connector
